mantidproject/mantid
# Adding plotting, fitting and results table into FDA

I have:
 - added plotting of the modulus of the freq spectrum to FDA
- added fitting into FDA
- added results table tab into FDA

The unit tests will be added as part of #26037



**To test:**
Open Muon Analysis 2 and make sure it works as before. My changes should not have altered it.

Open frequency domain analysis 
Load some data
Use the GUI.
Things to watch out for:
 - does the plot of freq update correctly?
 - does the default fitting selection make sense?
 - can you fit to the freq spec and make a results table?
 - does it work with PhaseQuad?
 - can you still do time domain analysis?


Fixes #25987  




---
Does not need to be in release notes as it will be covered in a later issue/PR
#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
